io-engine/src/core/snapshot.rs

```diff
@@ -28,6 +30,39 @@ impl SnapshotParams {
     }
 }
 
+/// VolumeSnapshot Parameters to list the snapshots.
```

Comment:
It looks like it's not Parameters to list snapshots but rather a snapshot descriptor that represents existing snapshot.